Library/Homebrew/utils/analytics.rb

```diff
@@ -207,13 +207,58 @@ def get_analytics(json, args:)
         end
       end
 
+      # This method is undocumented because it is not intended for general use.
+      # It relies on screen scraping some GitHub HTML that's not available as an API.
+      # This seems very likely to break in the future.
+      # That said, it's the only way to get the data we want right now.
+      def output_github_packages_downloads(formula, args:)
+        return unless args.github_packages_downloads?
+        return unless formula.core_formula?
+
+        escaped_formula_name = CGI.escape(GitHubPackages.image_formula_name(formula.name))
```

Comment:
`ERB::Util.url_encode` is technically more correct here.

Though we don't actually escape these anywhere else (including bottle manifest downloads), probably because the restrictions on what a name can be already block everything that needs escaping.

Reply:
@Bo98 Updated. An earlier version seemed to require escaping but I've removed it here and it still works so doesn't any more 🎉